pdns/dnsdist-tcp.cc

```diff
@@ -337,13 +337,14 @@ void* tcpClientThread(int pipefd)
 
 	string poolname;
 	int delayMsec=0;
+	uint32_t tempFailureTTL = std::numeric_limits<uint32_t>::max();
```

Comment:
I have the feeling it would be better to make `tempFailureTTL` a member of `DNSQuestion`, it would make it easier to expose it to Lua later, if we want to. We should probably do the same thing for `delayMsec` and `poolname`, but that's another story.